internal/node/core.go

```diff
@@ -30,6 +31,11 @@ func NewCore(repo IRepo) *Core {
 
 // CreateNode creates a new node
 func (c *Core) CreateNode(ctx context.Context, m *Model) error {
+	nodeOperationCount.WithLabelValues(env, "CreateNode").Inc()
+
+	startTime := time.Now()
+	defer nodeOperationTimeTaken.WithLabelValues(env, "CreateNode").Observe(time.Now().Sub(startTime).Seconds())
```

Comment:
time in ms than secs

Reply:
Going by this : https://prometheus.io/docs/practices/naming/
> ...should use base units (e.g. seconds, bytes, meters - not milliseconds, megabytes, kilometers). See below for a list of base units.

Reply:
changed